packages/cashier/src/pages/payment-agent/payment-agent-detail/payment-agent-detail.jsx

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import './payment-agent-detail.scss';
+
+const PaymentAgentDetail = ({ action, children, className, has_red_color, icon, is_link, title }) => {
+    const detail = Array.isArray(children) ? children : [children];
+    return (
+        <div
+            className={classNames('payment-agent-detail', {
+                [className]: !!className,
+            })}
+        >
+            {icon && (
+                <div className='payment-agent-detail__icon-wrapper'>
+                    <Icon icon={`IcCashier${icon}`} data_testid='dt_payment-agent-detail-icon' />
```

Comment:
for data_testid, let's go with camel_casing

Reply:
@vinu-deriv oh, yes, thx for notice